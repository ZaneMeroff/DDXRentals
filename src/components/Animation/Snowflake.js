import React from 'react';
import snowflake from '../../data/snowflake.png';

const Snowflake = (props) => {
  console.log(props.locationY);
  return (
    <img src={ snowflake } className='fade-in' style={{position:'absolute', top:props.locationX, left:props.locationY + 100}} alt='snowflake animation'/>
  )
}

export default Snowflake;
