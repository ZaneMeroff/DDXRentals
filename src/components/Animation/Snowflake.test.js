import React from 'react';
import ReactDOM from 'react-dom';
import Snowflake from './Snowflake';
import { shallow } from 'enzyme';

describe('Snowflake', () => {

  it('should be an instance of Snowflake', () => {
    const wrapper = shallow(<Snowflake />);
    expect(wrapper).toMatchSnapshot();
  });

});
