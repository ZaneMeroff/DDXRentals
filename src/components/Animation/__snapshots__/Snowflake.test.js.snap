// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snowflake should be an instance of Snowflake 1`] = `
<img
  alt="snowflake animation"
  className="fade-in"
  src="snowflake.png"
  style={
    Object {
      "left": NaN,
      "position": "absolute",
      "top": undefined,
    }
  }
/>
`;
